### DISCUSSION TITLE
后台权限管理系统 | 融客空间 | For Consilience

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#276] [General]
# 后台权限管理系统 | 融客空间 | For Consilience

这是一个NestJs + Vben Admin 编写的一款前后端分离的权限管理系统。

http://dev.wangc.site/blog/vben-admin/

**Comment by giserman001** [comment]
这个项目可以开源吗？

**Reply by kuizuo** [reply]
这个项目的原始代码来自 https://github.com/kuizuo/nest-vben-admin
